test/hotspot/jtreg/runtime/logging/ProtectionDomainVerificationTest.java

```diff
@@ -41,17 +41,29 @@ public static void main(String... args) throws Exception {
         ProcessBuilder pb = ProcessTools.createJavaProcessBuilder("-Xlog:protectiondomain=trace",
                                                                   "-Xmx128m",
                                                                   Hello.class.getName());
-        OutputAnalyzer out = new OutputAnalyzer(pb.start());
-        out.shouldContain("[protectiondomain] Checking package access");
-        out.shouldContain("[protectiondomain] pd set count = #");
+        new OutputAnalyzer(pb.start())
+        .shouldContain("[protectiondomain] Checking package access")
+        .shouldContain("[protectiondomain] pd set count = #")
+        .shouldHaveExitValue(0);
```

Comment:
Minor nit: checking the exit value first can be more informative in case of a crash.

Reply:
The patterns I see have that last. (?)